### ISSUE TITLE
Rename/rework test keyword to keep test data hidden

### ISSUE THREAD

**Issue by bfhealy** [ZwickyTransientFacility/scope#138]
The `test` kwarg in the training code (e.g. below) is misleading - when it is `False` (default), statistics on the test set will be computed and logged. It is good practice to run the intermediate iterations of our training without looking at the test scores. Changing this keyword to something like 'evaluate_test' and reworking the boolean logic could hide the test data from us until we are ready to review and report it.

https://github.com/ZwickyTransientFacility/scope/blob/ac22f211a188eb1736e35eb9c6c552a902735fb5/scope.py#L658
